pyunlocbox/tests/test_functions.py

```diff
@@ -413,37 +410,59 @@ def test_independent_problems(self):
         # Test all available functions.
         funcs = inspect.getmembers(functions, inspect.isclass)
         for func in funcs:
+            print('Testing ' + func[0])
```

Comment:
Please don't leave `print` statements in tests. It's useful for debugging but it makes the test log noisy. You can know where in the loop the test failed by wrapping the code in a [`with self.subTest(...):`](https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests) block.